renderlib/io/FileReaderTIFF.cpp

```diff
@@ -65,6 +65,33 @@ split(const std::string& s, char delim, std::vector<std::string>& elems)
   }
 }
 
+// multi lines split by newline
+// each line split by =
+std::map<std::string, std::string>
+splitToNameValuePairs(const std::string& s)
+{
+  std::vector<std::string> sl;
+  split(s, '\n', sl);
+
+  // split each string into name/value pairs,
+  // then look up as a map.
+  std::map<std::string, std::string> pairs;
+  for (int i = 0; i < sl.size(); ++i) {
+    std::vector<std::string> namevalue;
+    split(sl[i], '=', namevalue);
```

Comment:
Is the data well-posed, or do you need to worry about whitespace?

Reply:
An extra whitespace trim wouldn't hurt here.  I believe this data is only ever written out programmatically so it's unlikely to have extra whitespace.  The problem is that I don't have a whole host of files to test with.  For most of these file format things, my philosophy is wait till someone tells me their file won't load and then decide whether to fix it.